do-app-platform-op-cli/README.md

```diff
@@ -1,314 +1,285 @@
-# Deploy 1Password SCIM Bridge on DigitalOcean App Platform with 1Password CLI
+# Deploy 1Password SCIM Bridge on DigitalOcean App Platform using 1Password CLI
 
-This deployment example describes how to deploy 1Password SCIM bridge as an app on DigitalOcean's [App Platform](https://docs.digitalocean.com/products/app-platform/) service using [1Password CLI](https://developer.1password.com/docs/cli), the DigitalOcean command line interface ([`doctl`](https://docs.digitalocean.com/reference/doctl/)), and optionally the DigitalOcean [1Password Shell Plugin](https://developer.1password.com/docs/cli/shell-plugins/) (for Mac and Linux users).
+_Learn how to deploy 1Password SCIM Bridge on the [DigitalOcean App Platform](https://docs.digitalocean.com/products/app-platform/) service._
 
-Alternatively, you can [deploy 1Password SCIM Bridge to DigitalOcean App Platform using the DigitalOcean portal](https://support.1password.com/cs/scim-deploy-digitalocean-ap/).
+This deployment consists of two [resources](https://docs.digitalocean.com/glossary/resource/): A [service](https://docs.digitalocean.com/glossary/service/) for the for the SCIM bridge and an [internal service](https://docs.digitalocean.com/glossary/service/#internal-services) for Redis. There are a few benefits to deploying 1Password SCIM Bridge on DigitalOcean App Platform:
 
-The app consists of two [resources](https://docs.digitalocean.com/glossary/resource/): a [service](https://docs.digitalocean.com/glossary/service/) for the SCIM bridge container and an [internal service](https://docs.digitalocean.com/glossary/service/#internal-services) for Redis.
-
-## In this folder
-
-- [`README.md`](./README.md): the document that you are reading. 👋😃
-- [`op-scim-bridge.yaml`](./op-scim-bridge.yaml): an App Platform [app spec](https://docs.digitalocean.com/glossary/app-spec/) for 1Password SCIM bridge [templated with secret references](https://developer.1password.com/docs/cli/secrets-template-syntax) to load secrets from your 1Password account.
+- **Low cost:** For standard deployments, the service will host your SCIM bridge for ~$10 USD/month (as of March 2024). App Platform pricing is variable based on activity, and you can learn more on [DigitalOcean's pricing page](https://www.digitalocean.com/pricing/app-platform).
+- **Automatic DNS record management:** You don't need to manage a DNS record. App Platform automatically provides a unique one for your SCIM bridge domain.
+- **Automatic TLS certificate management:** App Platform automatically handles TLS certificate management on your behalf.
+- **Multiple deployment options:** The SCIM bridge can be deployed directly to DigitalOcean from your local terminal using this guide, or from the DigitalOcean Apps portal using the [support guide](https://support.1password.com/cs/scim-deploy-digitalocean-ap/). If you're using a custom deployment, cloning this repository is recommended.
```

Comment:
To avoid passive voice.

```suggestion
- **Multiple deployment options:** You can deploy the SCIM bridge directly to DigitalOcean from your local terminal using this guide, or from the DigitalOcean Apps portal using the [support guide](https://support.1password.com/cs/scim-deploy-digitalocean-ap/). If you're using a custom deployment, cloning this repository is recommended.
```